### ISSUE TITLE
Event sortstart not triggered

### ISSUE THREAD

**Issue by on-delete** [lukasoppermann/html5sortable#176]
I have three lists that are connected with each other. Also i have bind the sortstart event to every list but it is not triggered. 

```
$('#todolist, #inprogresslist, #donelist').sortable().bind('sortstart', function(e, ui){
    console.log("test");
});
$('#todolist, #inprogresslist').sortable({
    connectWith: 'connected'
});
$('#inprogresslist, #donelist').sortable({
    connectWith: 'connected2'
});

<td>
    <ul class="connected list" id="todolist">
        <li>Item 1</li>
        <li>Item 2</li>
        <li>Item 3</li>
    </ul>
</td>
<td>
    <ul class="connected list" id="inprogresslist">
        <li>Item 1</li>
        <li>Item 2</li>
    </ul>
</td>
<td>
    <ul class="connected list" id="donelist"></ul>
</td>
```

So in the console there should be the 'test' output if i start dragging on any list, but it doesn't. 

If i change the event to dragstart, it works.

```
$('#todolist, #inprogresslist, #donelist').sortable().bind('dragstart', function(e, ui){
    console.log("test");
});
```

Tested with Firefox and Chrome.

**Comment by lukasoppermann** [comment]
Hey @on-delete the problem is in how you initialize the sortables.

When you do this

```
$('#todolist, #inprogresslist, #donelist').sortable().bind('sortstart', function(e, ui){
    console.log("test");
});
$('#todolist, #inprogresslist').sortable({
    connectWith: 'connected'
});
$('#inprogresslist, #donelist').sortable({
    connectWith: 'connected2'
});
```

What you do is to setup the sortable with a bind, than reset it (read delete everything you just did) and set it up without a bind, than reset #inprogresslist again and set it up again.

TL;DR every time you call `sortable` on an element if it was a sortable before, it is reset first. So you need to move the bind into the definitions.
